metrics-operator/controllers/common/providers/dynatrace/dynatrace.go

```diff
@@ -42,14 +41,18 @@ type DynatraceData struct {
 // EvaluateQuery fetches the SLI values from dynatrace provider
 func (d *KeptnDynatraceProvider) EvaluateQuery(ctx context.Context, metric metricsapi.KeptnMetric, provider metricsapi.KeptnMetricsProvider) (string, []byte, error) {
 	baseURL := d.normalizeAPIURL(provider.Spec.TargetServer)
-	query := url.QueryEscape(metric.Spec.Query)
+
```

Comment:
out of curiosity does dql version of encoding work? I see we use values.encode() there

Reply:
just checked - DQL should be fine, as the complete query, including any time range filters etc. is part of the `query` parameter for the `query:execute` API, so we do not have multiple parameters here